content/grants/full-stop.md

```diff
@@ -0,0 +1,127 @@
+
+Even the [EFF has identified that, while it might not be necessary to delete a period tracking app, users should become familiar with the privacy policy of the app and make sure that it aligns with their risk profile.](https://www.eff.org/deeplinks/2022/06/should-you-really-delete-your-period-tracking-app) Further, they recommend considering all apps as a *permeable surface through which data about a potential pregnancy could leak* (suggesting that users should use E2E encrypted messaging, etc, as an additional security measure).
+At the risk of sounding flippant, Urbit fixes a great deal of this for users without interrupting their computing experience. Urbit provides an encrypted communication layer with identity primatives (affording identity assurances in comms). Urbit also acts as a personal server, avoiding the concern of corporate leaks of biometric data entirely. And yet, Urbit is available on a user's Phone as an interface for inputting data, maintaining the convenience of otherwise-centralized services. A period tracker on Urbit is simple solution to a very complex problem. It also presents some interesting development questions, reviewed below.
+
+# Overview
+Full Stop helps you track your period and period-related-accessory-phenomena on a secure, private, personal server called an urbit. **Full Stop** is accessed like a traditional phone application, available on major phone operating systems. **Full Stop** will include basic Push Notifications which will, given the need to pass through Apple's Great Filter, never include private information and should provide no indicia of a user's health conditions (or lack thereof) in their frequency or rate of occurrence.
+On first opening **Full Stop**, a user should enter a simple PIN, in addition to their `+code` required to log into their Urbit, generally. This pin provides a final layer of security, on top of their Phone's security, on top of Urbit's encryption and login requirements, as an additional precaution to ensure private access, only.
+After establishing their security PIN, Full Stop will proceed as with most period tracker applications:
+### Recording Data
+- **Period Data:** The user can record information about their period.
+    - Start and Stop dates
+    - Flow rate per day, during period
+    - Pain and Physical Symptom checklist per day
+- **Sexual Activity Data:** The user can record information about any sexual contact they may experience.
+    - Date of encounter
+    - Use or non-use of additional contraceptive protection (e.g. condoms)
+    - User's partner's @p (kidding)
+- **Additional BioMarkers:** The user can record period-adjacent data they may wish to record.
+    - Off-period spotting
+    - Birth Control Pill consumption tracker (possibly alerts, though we're considering the data leak possibilities through push notifications)
```

Comment:
```suggestion
    - Birth Control Pill consumption tracker
    - Term Birth Control application and renewal date tracker (e.g. IUD)
 - **Pregnancy Tracking:** The user can record information about their pregnancy, and track the weeks until their due date.
   - User can set a due date from the date of the start of the pregnancy or their projected due date.
   - The application will provide the user with a count of weeks since the start of their pregnancy.
- **Fertility Prediction:** The user can elect to view various projections of fertility windows based on their prior period data, as an aid for con(tra)ception.
   - Methods include timing methods and, later, application of basal body temp and cervical mucosal density data inputted by the user to further specify fertility windows 
```